docs/conf/eu/2016/news/thanks-for-coming.md

```diff
@@ -0,0 +1,64 @@
+:template: 2016/eu.html
+
+Thanks for coming to Write the Docs Europe 2016
+===============================================
+
+Hi everyone,
+
+Thanks for attending our third European Write the Docs conference in Prague! We're so grateful to our sponsors, speakers, volunteers, and attendees for making the conference possible. 
+
+We hope you had a wonderful time, and that you are as excited as we are about the beautiful community that is forming. Below we've included all of the resources you might need for staying in touch, like the Write the Docs Slack. We also have a short anonymous survey for you to fill out, so we can make the conference even better next year.
+
+## Tell us how we did
+
+Let us know how we can improve for future conferences and gatherings. We really appreciate your candid feedback!
+
+**Our three question survey is quick and easy, so [go fill it out](https://goo.gl/forms/RQE7qyaFvaMNP2Zo2).**
+
+## Speak, Sponsor, or Organize next year
+
+This is the end of our 2016 round of conferences,
+but that doesn't mean things are over!
+We'll be having another North American and European conference next year.
+
+We're always looking for folks who want help us organize the events.
+We have worked to document the [roles](http://www.writethedocs.org/organizer-guide/confs/event-roles/) that need to happen to put the conference together.
+We could use a few folks who are interested in lending a hand to make the event happen,
+so please let us know if that's you!
+
+We also depend on sponsors to make our events as magical as they are,
+while keeping ticket prices low.
+If you are curious about sponsoring Europe or North America next year,
+please contact us at [sponsorship@writethedocs.org](mailto:sponsorship@writethedocs.org).
+
+Also don't forget that inspiration and start thinking of your topic ideas for next year.
+We'll be opening our North American CFP in December,
+and Europe will be open in April next year.
```

Comment:
Yea, just seemed inconsistent to say "Portland and European" -- it works if we do "Portland and Prague", but I think keeping them both either city or geographical works.